author: Andres Salomon <dilinger@debian.org>
description: fix bindgen-related stuff

Also, the call to bindgen sets the path for libclang to
rust_bindgen_root, which is wrong. We're already passing
clang_base_path with the path to libclang, there's no reason that
we'd expect libclang to be in the same directory as bindgen. That
fix should probably go upstream.


Index: chromium-144.0.7524.0/build/rust/rust_bindgen.gni
===================================================================
--- chromium-144.0.7524.0.orig/build/rust/rust_bindgen.gni
+++ chromium-144.0.7524.0/build/rust/rust_bindgen.gni
@@ -19,7 +19,7 @@ if (host_os == "win") {
 
 # On Windows, the libclang.dll is beside the bindgen.exe, otherwise it is in
 # ../lib.
-_libclang_path = rust_bindgen_root
+_libclang_path = clang_base_path
 if (!use_chromium_rust_toolchain &&
     (current_cpu == "ppc64" || current_cpu == "s390x")) {
   _libclang_path = rust_sysroot_absolute + "/lib64"
